README.md

```diff
@@ -1,75 +1,77 @@
+ (contributions welcome! 🙂).
 
-## Features
-- Resigns and installs any app with your Apple ID
-- Installs apps over WiFi using AltServer
-- Refreshes apps periodically in the background to prevent them from expiring (when on same WiFi as AltServer)
-- Handles app updates directly through AltStore 
 
 ## Requirements
 - Xcode 11
-- iOS 12.2+ (AltStore)
-- macOS 10.14.4+ (AltServer)
+- iOS 12.2+ (SideStore)
+- macOS 10.14.4+ (TBD)
 - Swift 5+
-
-Why iOS 12.2+ and macOS 10.14.4+? Doing so allows me to distribute all AltStore apps without embedding Swift libraries inside them. This helps me afford bandwidth costs by reducing download sizes by roughly 30%, but also noticeably improves how long it takes to install/refresh apps with AltStore. If you're compiling AltStore and/or AltServer yourself, however, you should be able to lower their deployment targets to iOS 12.0 and macOS 10.14.0, respectively, without any issues.
+- Visual Studio Code
 
 ## Project Overview
 
-### AltStore
-AltStore is a just regular, sandboxed iOS application. The AltStore app target contains the vast majority of AltStore's functionality, including all the logic for downloading and updating apps through AltStore. AltStore makes heavy use of standard iOS frameworks and technologies most iOS developers are familiar with, such as:
-* Core Data
-* Storyboards/Nibs
-* Auto Layout
-* Background App Refresh
-* Network.framework (new in iOS 12)
-
-### AltServer
-AltServer is also just a regular, sandboxed macOS application. AltServer is significantly less complex than AltStore though, and for that reason consists of only a handful of files.
-
-### AltKit
-AltKit is a shared framework that includes common code between AltStore and AltServer.
+### SideStore
+SideStore is a alternative to Altstore and is a sandboxed iOS application like Altstore. The SideStore app target contains the vast majority of AltStore's functionality, including all the logic for downloading and updating apps through SideStore.
 
-### AltSign
-AltSign is my internal framework used by both AltStore and AltServer to communicate with Apple's servers and resign apps. For more info, check the [AltSign repo](https://github.com/rileytestut/altsign).
+### Netmuxd
+Netmuxd is a program that replaces Usbmuxd to be able to connect over a VPN reliably.  It is programmed in the Rust programming language and it is open source. You can set Netmuxd to be a hyper link like Netmuxd: [Netmuxd](https://github.com/jkcoxson/netmuxd)
 
 ### Roxas
-Roxas is my internal framework used across all my iOS projects, developed to simplify a variety of common tasks used in iOS development. For more info, check the [Roxas repo](https://github.com/rileytestut/roxas).
+Roxas is Riley Testuts internal framework from Altstore used across many of their iOS projects, developed to simplify a variety of common tasks used in iOS development. For more info, check the [Roxas repo](https://github.com/rileytestut/roxas).
```

Comment:
Testut's, Alt**S**tore

Reply:
fixed in https://github.com/SideStore/SideStore/pull/11/commits/40660e459e0d662eca5fb6ffcd284fa2e9338994